engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/BucketedPartitionedUpdateByManager.java

```diff
@@ -121,6 +115,12 @@ class BucketedPartitionedUpdateByManager extends UpdateBy {
             transformFailureListener = new TransformFailureListener(transformedTable);
             transformedTable.addUpdateListener(transformFailureListener);
             result.addParentReference(transformFailureListener);
+
```

Comment:
@rcaudy This moves initialization of the listener after it's dependencies are fully initialized.  Doesn't belong in this commit, but correcting a bad pattern (but not incorrect due the lock being held).